core/common/README.md

```diff
@@ -1,3 +1,3 @@
 # :core:common module
 
-![Dependency graph](../../docs/images/graphs/dep_graph_core_common.png)
+![Dependency graph](../../docs/images/graphs/dep_graph_core_common.svg)
```

Comment:
Does it make sense to show these graphs where there's only one module? There's no context so opening this page just shows an ellipse with the name of the module inside.

Reply:
I think it does still make sense to show modules with no dependencies, as it's explicitly showing that.